src/components/atoms/Buttons/Buttons.tsx

```diff
@@ -0,0 +1,18 @@
+import React, {FC} from 'react';
```

Comment:
file name will be just index.tsx and its style will be styles.ts

Reply:
For all the folders, should the sub files be named index and styles only?